msteams-platform/samples/virtual-assistant.md

```diff
@@ -273,10 +273,10 @@ The corresponding Virtual Assistant manifest file code snippet is shown in the f
                 "id": "searchQuery:<skill_id>",
                 "context": [ "compose", "commandBox" ],
                 "description": "Test command to run query",
-    ....
```

Comment:
Please verify. Why has this been removed?

Reply:
Hi @surbhigupta12 as shared by Habiba in the email trail, we found formatting issues in code which were all highlighted in red, also these dots were seen as extra dots in live docs, so we tried quickly fixing it, but it did not work, I added them back in the source file. Thank you.